src/client/components/header/search/Search.js

```diff
@@ -1,53 +1,77 @@
 import React from 'react';
-import styles from './Search.module.css';
-import PropTypes from 'prop-types';
+import './Search.module.css';
+import cn from 'classnames';
+import ToggleButton from '../../shared/toggleButton/ToggleButton';
 
-const activeStyle = {
-  backgroundColor: "#F65261"
-}
+class Search extends React.Component {
 
-const passiveStyle = {
-  backgroundColor: "rgba(66,66,66,0.68)"
-}
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchByTitle: {
+        label: "TITLE",
+        status: true,
+      },
+      searchByGengre: {
+        label: "GENGRE",
+        status: false,
+      }
+    }
+  }
 
+  clickTitle() {
+    this.setState((prevState) => {
+      let updState = JSON.parse(JSON.stringify(prevState));
```

Comment:
i don't see any reason to do this. This add only extra magic to your code. Just simply desctructure prevState